docker/README.md

```diff
@@ -8,11 +8,11 @@ Miaou development environment using Docker
 ### Linux
 
  - [Docker and Docker Compose](https://docs.docker.com/engine/installation/linux/)
- - Docker needs Hyper-V activated and purpose to activate it itself (restart needed). Beware : activating Hyper-V disables any other virtualisation softwares.
 
 ### Windows
 
  - [Docker and Docker Compose](https://docs.docker.com/engine/installation/windows/)
+ - Docker needs Hyper-V activated and purpose to activate it itself (restart needed). Beware : activating Hyper-V disables any other virtualisation softwares.
```

Comment:
Should be deleted, this is a Docker prerequisite.

Reply:
not a Docker prerequisite if you don't use `Docker for Windows`, but indeed 99.99% of docker users will use it, so the whole `Requirements` section seems overkill to me (or explicitly mention that we assume they will use `Docker for Windows` for Windows and `Docker for Mac` for Mac)... The `Linux` section remains useful only to tell users to install `docker-compose` which is not a default when installing docker on linux